config.go

```diff
@@ -43,6 +43,8 @@ type Config struct {
 		Partitioner PartitionerConstructor
 		// If enabled, successfully delivered messages will be returned on the Successes channel (default disabled).
 		AckSuccesses bool
+		// If enabled, messages that failed to deliver will be returned on the Errors channel, including error (default enabled).
+		AckErrors bool
```

Comment:
I dislike the name `AckErrors` but I don't have anything better :(

Maybe `Return.Errors` and `Return.Successes`?